web-extension/pow-worker.js

```diff
@@ -3,8 +3,6 @@ import { PoW } from '@vegaprotocol/crypto'
 // Independent PoW solver as a web worker to unblock the background thread
 // the requests are JSON-RPC to ease the client implementation
 globalThis.onmessage = async function(ev) {
-  // FIXME: this is a hack to make the tests work because the pow-worker.js file is being incorrectly bundled
-  // into the application when it should be requested on demand instead.
   if (ev.data?.method !== 'solve') return
```

Comment:
@dalebennett1992 @emilbayes This should now be able to be removed now that the app is not importing scripts it shouldn't.

Reply:
I still think it’s a good measure to have in place